packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,238 @@
+    data: () => ({
+        enabledCardTypes: [
+            'Header_Card',
+            'Promotion_Card_1',
+            'Promotion_Card_2',
+            'Recommendation_Card_1',
+            'Restaurant_FTC_Offer_Card',
+            'Voucher_Card_1',
+            'Anniversary_Card_1'
+        ],
+        title: 'Offers',
+        hasMounted: false,
+        contentCardsHaveLoaded: false,
+        error: undefined,
+        showContentCards: true,
+        tries: 0,
+        isModalOpen: false,
+        modalOngoingUrl: '',
+        loadingCard: { type: 'postOrder', count: 3 },
+        pushToDataLayer: () => {}
+    }),
+
+    computed: {
+        ...mapState(VUEX_MODULE_NAMESPACE_OFFERS, [
+            'brazeApiKey',
+            'globalUserId'
+        ]),
+
+        /**
+         * Determines the tenant based on the currently selected locale in order to choose correct translations
+         * @return {String}
+         **/
+        tenant () {
+            return {
+                'en-GB': 'uk',
+                'en-AU': 'au',
+                'en-NZ': 'nz',
+                'da-DK': 'dk',
+                'es-ES': 'es',
+                'en-IE': 'ie',
+                'it-IT': 'it',
+                'nb-NO': 'no'
+            }[this.locale] || 'uk';
+        }
+    },
+
+    /**
+     * Handles the setting of appboy on the window in order to prevent a race condition between internally
+     * and externally loaded versions of the braze sdk
+     */
+    mounted () {
+        this.mountTime = Date.now();
+        this.hasMounted = true;
+    },
+
+    methods: {
+        /**
+         * Maps given card type to component name
+         *
+         * @param type
+         * @return {string|boolean}
+         */
+        handleCustomCardType (type) {
+            switch (type) {
+                case 'Anniversary_Card_1':
+                case 'Voucher_Card_1':
+                    return 'VoucherCard';
+                case 'Restaurant_FTC_Offer_Card':
+                    return 'FirstTimeCustomerCard';
+                case 'Promotion_Card_1':
+                case 'Promotion_Card_2':
+                    return 'PromotionCard';
+                case 'Terms_And_Conditions_Card':
+                case 'Terms_And_Conditions_Card_2':
+                    return 'TermsAndConditionsCard';
+                default:
+                    break;
+            }
+            return false;
+        },
+
+        /**
+         * Consumes the number of cards received from braze by the content cards component
+         * @param {number} cards
+         */
+        cardsReceived (cards) {
+            if (cards || this.contentCardsHaveLoaded) {
+                this.logInfo(`Content cards loaded: ${cards}`, this.getLogPayload(true));
+            }
+        },
+
+        /**
+         * Handles a successful load of the content cards component
+         */
+        hasLoaded () {
+            this.contentCardsHaveLoaded = true;
+            this.tries = 0;
```

Comment:
`tries` doesn't have a purpose anymore if we're not attempting to reload on error.
```suggestion
```